src/components/icons/custom-icons/Community.stories.tsx

```diff
@@ -0,0 +1,17 @@
+import { Meta, Story } from '@storybook/react';
+import React from 'react';
+
+import { CommunityIcon } from '../';
+import { SvgProps } from '../types';
+
+export default {
+  component: CommunityIcon,
+  title: 'Icons/CustomIcons/CommunityIcon',
```

Comment:
💬 For this, and the other icons, I think it would be better to just have them flatly aligned listed in the icons story level

```suggestion
  title: 'Icons',
```

Reply:
Done!